server/modelpath_test.go

```diff
@@ -157,3 +158,67 @@ func TestParseModelPath(t *testing.T) {
 		})
 	}
 }
+
+func TestMigrateRegistryDomain(t *testing.T) {
+	manifests := []string{
+		filepath.Join("registry.ollama.ai", "library", "mistral", "latest"),
+		filepath.Join("registry.ollama.ai", "library", "llama3", "7b"),
+		filepath.Join("ollama.com", "library", "llama3", "7b"),
+		filepath.Join("localhost:5000", "library", "llama3", "13b"),
+	}
+
+	models := t.TempDir()
+	t.Setenv("OLLAMA_MODELS", models)
+	envconfig.LoadConfig()
```

Comment:
This is _extremely_ error prone. First local run migrated my local models directory since it didn't load the new path